packages/trader/src/App/Containers/Routes/routes.tsx

```diff
@@ -9,11 +8,29 @@ import getRoutesConfig from 'App/Constants/routes-config';
 import { observer, useStore } from '@deriv/stores';
 import { useTraderStore } from 'Stores/useTraderStores';
 
-const checkRoutingMatch = (route_list, path) => {
+type TMatchPattern = { from: Array<string | undefined>; to: string[] };
```

Comment:
Maybe it's better to stick to one code style: Array<some_type> or some_type[]?

Reply:
@kate-deriv at least on one line, yes, i'll update this, thank you  :)